### PR TITLE
XW-1856 | divider length on desktop; full site link appearance

### PR DESCRIPTION
**Pull request by Warkot**
Applied designer's feedback.

https://wikia-inc.atlassian.net/browse/XW-1856

@Wikia/x-wing

### CODE REVIEW

File: components/_global-footer.scss [hunk 1]
```diff
@@ -88,11 +88,28 @@
...
+		&.wds-has-no-padding {
+			padding: 0;
+		}
+	}
+
+	// @todo To be moved to a separate component
+	&__button-link {
+		display: inline-block;
+		font-weight: bold;
+		letter-spacing: .3px;
```
Comment (hakubo) [inline components/_global-footer.scss]: that should be defined in global styles, should it?
why do we define it only for button link?

File: components/_global-footer.scss [hunk 2]
```diff
@@ -88,11 +88,28 @@
...
 
 	&__bottom-bar-row {
+		margin: 0 auto;
 		padding: 16px 0;
 
 		&.wds-has-border-top {
 			border-top: 1px solid $color-slate-gray;
 		}
+
+		&.wds-has-no-padding {
```
Comment (hakubo) [inline components/_global-footer.scss]: if we  did the opposite we would not have to override padding property.

File: components/_global-footer.scss (hunk 1 above)
Comment (Warkot) [inline components/_global-footer.scss]: I copied it from the "Start a Wikia" button and we should define it for all buttons as it's defined here https://zpl.io/aohS8, when creating a button component in DS.

File: components/_global-footer.scss (hunk 2 above)
Comment (Warkot) [inline components/_global-footer.scss]: My intuition told me that the default row should be padded. The row with just a button seemed like an custom exception (yeah, I know there are only two rows 😝 ). And the option you're proposing might look like that:
```
	&__bottom-bar-row {
		margin: 0 auto;
		…
		&.wds-has-padding {
			padding: 16px 0;
		}
	}
```
…and it actually looks pretty damn good, now when I look at it. 👍

**Comment by Warkot** [comment]
LGTM from @hakubo ;)

**Comment by rogatty** [comment]
LGTM

**Pull request merged**